packages/text/src/TextMetrics.ts

```diff
@@ -643,15 +644,16 @@ export class TextMetrics
             fontSize: 0,
         };
 
-        const canvas = TextMetrics._canvas;
         const context = TextMetrics._context;
 
         context.font = font;
-
         const metricsString = TextMetrics.METRICS_STRING + TextMetrics.BASELINE_SYMBOL;
-        const width = Math.ceil(context.measureText(metricsString).width);
+
+        const textMetrics = context.measureText(metricsString);
+        const canvas = TextMetrics._canvas;
```

Comment:
slightly pedantic, but if you move
this const canvas = TextMetrics._canvas;
line back to where it was, about the  const context = TextMetrics._context; line, it reduces the diff down to just the changes required, and reads a bit better.

Reply:
Yah sure!